docs/customization/model.rst

```diff
@@ -23,11 +23,17 @@ To give you an idea of some purposes of models customizing have a look at a few
 * Add ``icon`` to the ``PaymentMethod``
 
 And of course many similar operations limited only by your imagination.
+
 Let's now see how you should perform such customizations.
 
 How to customize a Model?
 -------------------------
 
+.. tip::
+
+    You can browse the code of this example on Github together with tests, twig, form etc.
+    on the `sylius/customization/how_to_customize_a_model <https://github.com/Sylius/Customizations/pull/2>`_
```

Comment:
Won't it use `sylius/customization/how_to_customize_a_model` as a link title?

Reply:
Yes, it will and I agree it will look bad. :<